packages/core/src/global/grid.scss

```
@@ -0,0 +1,70 @@
+.tds-grid-container {
+  display: flex;
+  justify-content: center;
+}
+
+.tds-grid-fixed,
+.tds-grid-fluid {
+  display: grid;
+  width: calc(100% - 32px);
+  padding: 0 16px;
+  grid-template-columns: repeat(6, 1fr);
+  grid-template-rows: repeat(2, 1fr);
+  grid-gap: 16px;
+}
+
+@media screen and (min-width: 416px) {
+  .tds-grid-fixed {
+    width: calc(416px - 32px);
+  }
+
+  .tds-grid-fluid {
+    grid-template-columns: repeat(12, 1fr);
+    grid-template-rows: 1fr;
+  }
+}
+
+@media screen and (min-width: 604px) {
+  .tds-grid-fixed {
+    width: calc(604px - 32px);
+    grid-template-columns: repeat(12, 1fr);
+    grid-template-rows: 1fr;
+  }
+}
```

```suggestion
@media screen and (min-width: 604px) {
  .tds-grid-fixed {
    width: calc(604px - 32px);
    grid-template-columns: repeat(12, 1fr);    
  }
}
```

Same here. 
`grid-template-rows: 1fr;`